test/SlowTests/Issues/RavenDB_13293.cs

```diff
@@ -123,6 +123,7 @@ public async Task CanPassNodeTagToRestoreBackupOperation()
                         MentorNode = node
                     };
                     var result = await store.Maintenance.SendAsync(new UpdatePeriodicBackupOperation(backupConfig));
+                    await WaitForRaftIndexToBeAppliedInCluster(result.RaftCommandIndex, TimeSpan.FromSeconds(15));
```

Comment:
We `WaitForIndexNotification` after the command was completed.
Do we need this here? @karmeli87

Reply:
It was completed on the node you were talking to.
If later you are fetching the info from a different one, it might not be there..